Wikipedia/Code/WMFKeychainCredentials.swift

```diff
@@ -104,79 +79,35 @@ struct WMFKeychainCredentials {
             kSecAttrAccount as String : key as AnyObject
         ]
     }
-    
-    // MARK: Getting values from keychain
-    
-    private func tryGetString(forKey key: String) -> String? {
-        return try? string(forKey: key)
-    }
-    
-    private func tryGetValue(forKey key: String) -> Any? {
-        return try? value(forKey: key)
-    }
-    
-    private func string(forKey key: String) throws -> String {
-        let queryDictionary = matchQuery(forKey: key) as CFDictionary
```

Comment:
It's been a while so I don't recall details, but was using `matchQuery` here important?

iirc I added this [comment](https://github.com/wikimedia/wikipedia-ios/pull/2354/files#diff-0f72898e428341fe29ccb13e091ccfe4L9) as a reminder of why the internals were structured as they were, but also unsure if your change impacts what the comment refers to...

Reply:
This is a revert on `WMFKeychainCredentials.swift` back to where it was before the addition of the event logging values. `matchQuery` was added as a convenience method and this reversion removes that convenience method and has it inline. It's possible we want to keep this convenience method, but I just brought over the file from the 5.8.1 release.